src/js/constants.js

```diff
@@ -0,0 +1 @@
+export default API_URL = 'https://api.stuyspec.xyz/articles';
```

Comment:
Specify STUY_SPEC_API, cause we might have other APIs. Also omit the articles, so you can reuse the URL